src/api/public/apidocs/components/parameters/meta.yaml

```diff
@@ -0,0 +1,10 @@
+in: query
+name: meta
+schema:
+  type: string
+  enum:
+  - 1
+  - 0
+  default: 0
+description: Request meta information instead of the normal info.
```

Comment:
Neither "meta" information nor "normal" information are a thing in this documentation. I think we should explain the difference.

Reply:
How about we call it [metadata](https://en.wiktionary.org/wiki/metadata#Noun) to keep it generic?

Reply:
according to `ack` we call it project meta file / package meta file.

Reply:
I renamed the file to `package_meta` and made the description more specific.